utils/fibery/renderer.module.scss

```diff
@@ -0,0 +1,242 @@
+@import 'styles/responsiveness';
+@import 'styles/variables';
+
+/* stylelint-disable no-descending-specificity */
+
+$block-spacing: 60px;
+
+.article-ordered {
```

Comment:
what are you think about each class in this file - own module?

Reply:
it's either separate files or we can't use `bem` lib